### ISSUE TITLE
React: catch duplicate attributes

### ISSUE THREAD

**Issue by rstacruz** [feross/standard#317]
`no-dupe-keys` catches these duplicates in an Object:

```js
console.log({ hi: 1, hi: 1 })
```

but not these attributes in a React element:

```js
import React from 'react'

window.hi(<div hi='1' hi='1'></div>)
```

**Comment by dtinth** [comment]
Relevant rule: [react/jsx-no-duplicate-props](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-duplicate-props.md).

**Comment by dcousens** [comment]
See https://github.com/feross/eslint-config-standard-react/pull/8

**Comment by Flet** [comment]
Cool this has been published in `eslint-config-standard-react@1.2.1` which was added to `standard@5.4.0` via https://github.com/feross/standard/commit/80cc9548c8881425e9ebc0b886428dc1424a510b

Closing!
